src/s2wasm-main.cpp

```diff
@@ -80,7 +80,7 @@ int main(int argc, const char *argv[]) {
   AllocatingModule wasm;
   uint64_t globalBase = options.extra.find("global-base") != options.extra.end()
                           ? std::stoull(options.extra["global-base"])
-                          : 1;
+                          : 0;
```

Comment:
actually let's keep it 1, then plus alignment it will be 16 (or the proper amount)?